backend/docs/migration.py

```diff
@@ -0,0 +1,126 @@
+# This script interacts with the Kubeflow Pipelines (KFP) API to:
```

Comment:
Should probably add the Apache License header to remain consistent with the rest of our Python scripts